.eslintrc.cjs

```diff
@@ -439,6 +439,19 @@ module.exports = {
         'unicorn/prefer-top-level-await': 'error',
       },
     },
+    {
+      files: [
+        'e2e/coverage-report/__mocks__/sumDependency.js',
+        'e2e/require-main-after-create-require/empty.js',
+        'packages/create-jest/src/__tests__/__fixtures__/**/*',
+        'packages/jest-core/src/__tests__/**/*',
+        'packages/jest-haste-map/src/__tests__/test_dotfiles_root/**/*',
+        'packages/jest-resolve/src/__mocks__/**/*',
+      ],
+      rules: {
+        'unicorn/no-empty-file': 'off',
+      },
+    },
```

Comment:
snuck this in as well - there are none of the current empty files that aren't empty on purpose